WcaOnRails/config/routes.rb

```diff
@@ -143,6 +143,7 @@
       get '/search/regulations' => 'api#regulations_search'
       get '/users/:id' => 'api#show_user_by_id', constraints: { id: /\d+/ }
       get '/users/:wca_id' => 'api#show_user_by_wca_id'
+      get '/registrations/:competition_id' => 'api#show_registrations_by_comp_id'
```

Comment:
This feels strange to me... I'd expect `/registrations/:registration_id` to be an endpoint to get details about 1 specific registration.

How about nesting this under `/competitions` instead? Something like `/competitions/:competition_id/registrations`.